packages/react-dom/src/__tests__/ReactDOMOption-test.js

```diff
@@ -41,9 +42,10 @@ describe('ReactDOMOption', () => {
     expect(() => {
       node = ReactTestUtils.renderIntoDocument(el);
     }).toWarnDev(
-      '<div> cannot appear as a child of <option>.\n' + '    in option (at **)',
+      'Only strings and numbers are supported as <option> children.\n' +
```

Comment:
I just reused the same warning message we have in SSR. Seems to make more sense than the DOM nesting warning I tried to use before.